docs/en/index.md

```diff
@@ -0,0 +1,341 @@
+
+You can disable this with YAML configuration:
+
+```yml
+# app/_config/elements.yml
+DNADesign\Elemental\Controllers\ElementController:
+  include_default_styles: false
+```
+
+### Implementing search
+
+The Elemental module comes with an indexer for Solr (via the
+[silverstripe-fulltextsearch module](https://github.com/silverstripe/silverstripe-fulltextsearch)). You can enable
+this index in your search engine to ensure that a page's elemental area content is indexed under the page's data.
+
+For information on configuring Solr please see [the fulltextsearch documentation](https://github.com/silverstripe/silverstripe-fulltextsearch).
+
+> [!NOTE]
+> If using this indexer, be aware that HTML tags will be stripped from the content before it is indexed.
+> The Solr search results may add in emphasis tags or other formatting around matched key words, so you may need
+> to allow unescaped HTML in your search results template. You should use the `$Excerpt` property (see
+> `SolrIndex::search` for more) to display the relevant search matches.
+
+### Disabling CMS content search
+
+When installing this module, the page model admin search will look for a term in the entire content of each elemental pages.
+This is done by rendering each page, which can be resource hungry and make the search timeout.
+
+You can disable this with YAML configuration:
+
+```yml
+# app/_config/elements.yml
+DNADesign\Elemental\Controllers\ElementSiteTreeFilterSearch:
+  search_for_term_in_content: false
+```
+
+### Usage of gridField
+
+This module used to use GridField to create and update Elements in the CMS. This has now been largely succeeded by a JavaScript interface via React. However elements that are marked as being incompatible with in-line editing will still use the GridField method.
+
+## Building the elemental frontend assets
+
+This module uses the [Silverstripe CMS Webpack module](https://github.com/silverstripe/webpack-config), and inherits
+things from the core Silverstripe CMS modules, such as a core variable sheet and JavaScript components.
+
+When making changes to either the SASS or JavaScript files, ensure you change the source files in `client/src/`.
+
+You can have [yarn](https://yarnpkg.com/en/) watch and rebuild delta changes as you make them (for development only):
+
+```bash
+yarn watch
+```
+
+When you're ready to make a pull request you can rebuild them, which will also
+minify everything. Note that `watch` will generate source map files which you
+shouldn't commit in for your final pull request. To minify and package:
+
+```bash
+yarn build
+```
+
+You'll need to have [yarn installed](https://yarnpkg.com/en/docs/install)
+globally in your command line.
+
+> [!NOTE]
+> If adding or modifying colours, spacing, font sizes etc. please try
+> and use an appropriate variable from the silverstripe/admin module if available.
+
+## Integration with other modules
+
+- [Multiple languages with tractorcow/silverstripe-fluent](docs/en/advanced_setup.md)
```

Comment:
```suggestion
```
This is linked to in the `[CHILDREN]` block below.

Reply:
Removed